docs/src/main/sphinx/sql/show-stats.rst

```diff
@@ -13,18 +13,24 @@ Synopsis
 Description
 -----------
 
-Returns approximated statistics for the named table or for the results of a query.
+Returns approximated statistics for the named table or for the results of a
+query. Returns ``NULL`` for any statistics that are not populated or
+unavailable on the data source.
 
-Statistics are returned for each column, along with a summary row.
+Statistics are returned as a row for each column, plus a summary row for
+the table (identifiable by a ``NULL`` value for ``column_name``). The following
+table lists the returned columns and what statistics they represent. Any
+additional statistics collected on the data source, other than those listed
+here, are not included.
 
-==========================  =============================================================
-Column                      Description
-==========================  =============================================================
-``column_name``             The name of the column (``NULL`` for the summary row)
-``data_size``               The total size in bytes of all of the values in the column
-``distinct_values_count``   The number of distinct values in the column
-``nulls_fractions``         The portion of the values in the column that are ``NULL``
-``row_count``               The number of rows (only returned for the summary row)
-``low_value``               The lowest value found in this column (only for some types)
-``high_value``              The highest value found in this column (only for some types)
-==========================  =============================================================
+==========================  ============================================================= =================================
+Column                      Description                                                   Notes
+==========================  ============================================================= =================================
+``column_name``             The name of the column                                        ``NULL`` in the table summary row
+``data_size``               The total size in bytes of all of the values in the column    ``NULL`` in the table summary row. Available for columns of textual types (``CHAR``, ``VARCHAR``, etc)
```

Comment:
connectors can report this for any type. are we sure this won't show up for non-textual types?

also, what "etc" stands for here?

also, what about structural types? for example, array or something can have `data_size` reported as well